yb-voyager/cmd/liveMigrationReportCommand.go

```diff
@@ -252,7 +255,10 @@ func updateExportedEventsCountsInTheRow(row *rowData, tableName string, schemaNa
 }
 
 func getFinalRowCount(row rowData) int64 {
-	return row.SnapshotRowCount + row.InsertsIn + row.InsertsOut - row.DeletesIn - row.DeletesOut
+	if row.DBType == "Source" {
```

Comment:
could this just be exportedRows + importedRows + insertsIn + insertsOut - deletesIn - deletesOut?

Reply:
yes not a problem though but I thought to keep it like that as this will be ideal and appropriate.

Reply:
yeah, seems fine either way!